core/src/main/java/com/graphhopper/routing/util/DataFlagEncoder.java

```diff
@@ -156,6 +164,18 @@ public int defineWayBits(int index, int shift) {
         carBwdMaxspeedEncoder = new EncodedDoubleValue("car bwd maxspeed", shift, speedBits, speedFactor, 0, maxPossibleSpeed, true);
         shift += carBwdMaxspeedEncoder.getBits();
 
+        /* Value range: [3.0m, 5.4m] */
```

Comment:
In many cases, the value is too big for the range as it implies something else rather than road restrictions. For instance, a bridge section over a river has 'height' attribute set to 60m. This value is actually the height from the river ground to the bridge. 

Btw, I tested with germany OSM and there were 77 exceptions due to value being too large.

java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: weight restriction value too large for encoding: 1000000, maxValue:100
java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 16, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 24, maxValue:12
java.lang.IllegalArgumentException: weight restriction value too large for encoding: 12000, maxValue:100
java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: width restriction value too large for encoding: 50, maxValue:6
java.lang.IllegalArgumentException: width restriction value too large for encoding: 8, maxValue:6
java.lang.IllegalArgumentException: height restriction value too large for encoding: 50, maxValue:12
java.lang.IllegalArgumentException: width restriction value too large for encoding: 10, maxValue:6
java.lang.IllegalArgumentException: width restriction value too large for encoding: 10, maxValue:6
java.lang.IllegalArgumentException: width restriction value too large for encoding: 10, maxValue:6
java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: width restriction value too large for encoding: 7, maxValue:6
java.lang.IllegalArgumentException: height restriction value too large for encoding: 70, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 16, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: width restriction value too large for encoding: 8, maxValue:6
java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: width restriction value too large for encoding: 30, maxValue:6
java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 16, maxValue:12
java.lang.IllegalArgumentException: width restriction value too large for encoding: 9, maxValue:6
java.lang.IllegalArgumentException: weight restriction value too large for encoding: 5000, maxValue:100
java.lang.IllegalArgumentException: height restriction value too large for encoding: 16, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 16, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 16, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: weight restriction value too large for encoding: 9000, maxValue:100
java.lang.IllegalArgumentException: width restriction value too large for encoding: 8, maxValue:6
java.lang.IllegalArgumentException: width restriction value too large for encoding: 8, maxValue:6
java.lang.IllegalArgumentException: width restriction value too large for encoding: 8, maxValue:6
java.lang.IllegalArgumentException: height restriction value too large for encoding: 100, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 50, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 50, maxValue:12
java.lang.IllegalArgumentException: width restriction value too large for encoding: 10, maxValue:6
java.lang.IllegalArgumentException: width restriction value too large for encoding: 8, maxValue:6
java.lang.IllegalArgumentException: width restriction value too large for encoding: 8, maxValue:6
java.lang.IllegalArgumentException: width restriction value too large for encoding: 7, maxValue:6
java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: weight restriction value too large for encoding: 30000, maxValue:100
java.lang.IllegalArgumentException: width restriction value too large for encoding: 7, maxValue:6
java.lang.IllegalArgumentException: width restriction value too large for encoding: 30, maxValue:6
java.lang.IllegalArgumentException: width restriction value too large for encoding: 12, maxValue:6
java.lang.IllegalArgumentException: width restriction value too large for encoding: 10, maxValue:6
java.lang.IllegalArgumentException: height restriction value too large for encoding: 220, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 50, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 100, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 60, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 50, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 50, maxValue:12
java.lang.IllegalArgumentException: width restriction value too large for encoding: 8, maxValue:6
java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: width restriction value too large for encoding: 8, maxValue:6
java.lang.IllegalArgumentException: width restriction value too large for encoding: 8, maxValue:6
java.lang.IllegalArgumentException: width restriction value too large for encoding: 8, maxValue:6
java.lang.IllegalArgumentException: width restriction value too large for encoding: 30, maxValue:6
java.lang.IllegalArgumentException: width restriction value too large for encoding: 8, maxValue:6
java.lang.IllegalArgumentException: width restriction value too large for encoding: 8, maxValue:6
java.lang.IllegalArgumentException: width restriction value too large for encoding: 10, maxValue:6
java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 100, maxValue:12
java.lang.IllegalArgumentException: width restriction value too large for encoding: 8, maxValue:6
java.lang.IllegalArgumentException: width restriction value too large for encoding: 8, maxValue:6
java.lang.IllegalArgumentException: height restriction value too large for encoding: 60, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 30, maxValue:12
java.lang.IllegalArgumentException: height restriction value too large for encoding: 279, maxValue:12
java.lang.IllegalArgumentException: width restriction value too large for encoding: 50, maxValue:6
tuan@tuan-VirtualBox:/mnt/data/transtech/geo-engine$ cat maven.log | grep "too large" | wc -l
77
tuan@tuan-VirtualBox:/mnt/data/transtech/geo-engine$ cat maven.log | grep "too large" | grep "height" | wc -l
41
tuan@tuan-VirtualBox:/mnt/data/transtech/geo-engine$ cat maven.log | grep "too large" | grep "weight" | wc -l
5
tuan@tuan-VirtualBox:/mnt/data/transtech/geo-engine$ cat maven.log | grep "too large" | grep "width" | wc -l
31